src/components/MagicCodeInput.js

```diff
@@ -120,9 +119,6 @@ function MagicCodeInput(props) {
             focusMagicCodeInput();
```

Comment:
This will set the input to an empty string. I think we can remove the resetFocus method now as it's same as focus

Reply:
I think I agree that we don't need an extra method now, I just committed the changes.